pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -177,11 +177,24 @@ pub mod pallet {
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
 		/// Dispatch the given call as a sub_type of pay_with_capacity. Calls dispatched in this
-		/// fashion, if allowed, will pay with Capacity,
+		/// fashion, if allowed, will pay with Capacity
+		// The weight calculation is a temporary adjustment because overhead benchmarks do not account
+		// for capacity calls.  We count reads and writes for a pay_with_capacity call,
+		// then subtract one of each for regular transactions since overhead benchmarks account for these.
+		//	 Storage: Msa PublicKeyToMsaId (r:1)
+		//   Storage: Capacity CapacityLedger(r:1, w:2)
+		//   Storage: Capacity CurrentEpoch(r:1) ? maybe cached in on_initialize
+		//   Storage: System Account(r:1)
+		//   Total (r: 4-1=3, w: 2-1=1)
 		#[pallet::call_index(0)]
 		#[pallet::weight({
+		use frame_support::weights::constants::RocksDbWeight;
 		let dispatch_info = call.get_dispatch_info();
-		(< T as Config >::WeightInfo::pay_with_capacity().saturating_add(dispatch_info.weight), dispatch_info.class)
+		let capacity_overhead = RocksDbWeight::get().reads(2)
+			.saturating_add(
+				RocksDbWeight::get().writes(1)
+			);
+		(< T as Config >::WeightInfo::pay_with_capacity().saturating_add(capacity_overhead), dispatch_info.class)
```

Comment:
you should add `dispatch_info.weight` to this

Reply:
Forgot that, thanks